stdlib/public/core/Arrays.swift.gyb

```diff
@@ -1412,6 +1416,7 @@ extension Array {
   ///
   /// - Complexity: O(`self.count`) if the array is bridged,
   ///   otherwise O(1).
+  @discardableResult
   public mutating func popLast() -> Element? {
```

Comment:
I'm not sure why, but the `popLast` here differs in "discardability" from the `popLast` in Collection.swift, where it has `@warn_unused_result`.